### PR TITLE
add first skeleton for product edit

### PR DESCRIPTION
**Pull request by and-wolf**

### CODE REVIEW

File: src/MerchantFrontend/src/app/views/merchant-products/merchant-product.resolver.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export class MerchantProductResolver implements Resolve<Product> {
   }
 
   resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<Product> {
-
+    console.log('resolver');
```
Comment (FloBWer) [inline src/MerchantFrontend/src/app/views/merchant-products/merchant-product.resolver.ts]: are the logs required here?

File: src/MerchantFrontend/src/app/views/merchant-products/merchant-products-detail/merchant-products-detail.component.ts [hunk 2]
```diff
@@ -15,9 +16,10 @@ export class MerchantProductsDetailComponent implements OnInit, OnDestroy {
   // Subscription
   private subResolver: Subscription;
 
-  constructor(private activeRoute: ActivatedRoute) {
+  constructor(private activeRoute: ActivatedRoute, private merchantApiService: MerchantApiService) {
     // Get resolved product from route
     this.subResolver = this.activeRoute.data.subscribe(value => {
+      console.log(value);
```
Comment (FloBWer) [inline src/MerchantFrontend/src/app/views/merchant-products/merchant-products-detail/merchant-products-detail.component.ts]: is that log requiered here?

File: src/MerchantFrontend/src/app/views/merchant-products/merchant-product.resolver.ts (hunk 1 above)
Comment (and-wolf) [reply src/MerchantFrontend/src/app/views/merchant-products/merchant-product.resolver.ts]: deleted

File: src/MerchantFrontend/src/app/views/merchant-products/merchant-products-detail/merchant-products-detail.component.ts (hunk 2 above)
Comment (and-wolf) [reply src/MerchantFrontend/src/app/views/merchant-products/merchant-products-detail/merchant-products-detail.component.ts]: deleted

File: src/MerchantFrontend/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,12064 @@
+{
```
Comment (FloBWer) [inline src/MerchantFrontend/package-lock.json]: belongs this file to the pr?

File: src/MerchantFrontend/angular.json [hunk 4]
```diff
@@ -2,7 +2,8 @@
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```
Comment (FloBWer) [inline src/MerchantFrontend/angular.json]: belongs this file to the pr?

Comment (and-wolf) [reply src/MerchantFrontend/angular.json]: No. Files are deleted

**Pull request merged**